roles/openshift_health_checker/openshift_checks/package_availability.py

```diff
@@ -0,0 +1,69 @@
+from openshift_checks import OpenShiftCheck, OpenShiftCheckException
+from openshift_checks.mixins import NotContainerized
+
+
+class PackageAvailability(NotContainerized, OpenShiftCheck):
+    """Check that required RPM packages are available."""
+
+    name = "package_availability"
+    tags = ["preflight"]
+
+    def run(self, tmp, task_vars):
+        try:
+            rpm_prefix = task_vars["openshift"]["common"]["service_type"]
+        except (KeyError, TypeError):
+            raise OpenShiftCheckException("'openshift.common.service_type' is undefined")
```

Comment:
One thing that I worry about, is that the checks are being run similar to modules, but they have access to variables that are not available to normal modules because task_vars are being passed through from the action plugin...  

I just worry that we are blending abstractions a bit too much here.